.github/workflows/build.yml

```diff
@@ -24,20 +24,23 @@ jobs:
   strategy:
     matrix:
       image:
-        - "quay.io/pypa/manylinux2014_x86_64"
+        - "quay.io/pypa/manylinux_2_28_x86_64"
```

Comment:
the reason I used the older image was because twine gave me some cryptic Python compatibility messages on the wheels when I used this newer one.  but if we know for sure that doesn't matter for our deployments, this is a good change

Reply:
I didn't attempt the upload so don't know what twine might show later, however all the GitHub actions fail on the older manylinux2014 build now, due to these now requiring node20.